README.md

```diff
@@ -5,9 +5,46 @@ Cuts out OpenStreetMap data, processes it to geodata and converts it to typical
 
 Website: http://osmaxx.hsr.ch/
 
-
 ## Development
-See https://github.com/geometalab/osmaxx-docs for documentations and `developmentEnvironment/` for instructions.
+See https://github.com/geometalab/osmaxx-docs for documentations and `/docs/development.md` for instructions.
+
+
+## Run it locally on Linux
+
+### Setup
+To run this project locally, you need docker and docker-compose installed 
+(https://docs.docker.com/installation/ubuntulinux/ and https://docs.docker.com/compose/install/).
+
+Then run 
+
+`./setup-containers.sh`
+
+### Running the project
+From now on you can start the project using docker compose:
+
+`docker-compose up webapp`
+
+### Running things inside the container
+`docker-compose run webapp <command>`
+
+For example, to run tests, you can use:
+
+`docker-compose run webapp python3 manage.py test`
```

Comment:
I assume we don't use `virtualenv` in the containers, because each container will host at most one python application, anyway, and is thus itself isolation enough.